src/core/cgroup.h

```diff
@@ -261,6 +261,16 @@ typedef enum CGroupIOAccountingMetric {
         _CGROUP_IO_ACCOUNTING_METRIC_INVALID = -EINVAL,
 } CGroupIOAccountingMetric;
 
+typedef enum CGroupMemoryAccountingMetric {
+        CGROUP_MEMORY_PEAK,
+        CGROUP_MEMORY_SWAP_PEAK,
+        CGROUP_MEMORY_SWAP_CURRENT,
+        CGROUP_MEMORY_ZSWAP_CURRENT,
+        _CGROUP_MEMORY_ACCOUNTING_METRIC_MAX,
+        _CGROUP_MEMORY_ACCOUNTING_METRIC_CACHED_LAST = CGROUP_MEMORY_SWAP_PEAK,
```

Comment:
doesn't matter, but I'd move this right next to CGROUP_MEMORY_SWAP_PEAK in the table up, so that people see this as the boundary that it is in the list of values, and when they add something new, they think about whether it should be placed before or after.

Reply:
maybe a nice brief comment what this is about would be good too